README.md

```diff
@@ -1,4 +1,18 @@
-# cacher [![codecov.io](http://codecov.io/github/kirillseva/cacher/coverage.svg?branch=master)](http://codecov.io/github/kirillseva/cacher?branch=master)[![Build Status](https://travis-ci.org/kirillseva/cacher.svg)](https://travis-ci.org/kirillseva/cacher)
+# cacher
```

Comment:
Note to self: We'll have to remember to remove this from the README after merging upstream.

Reply:
Could we just ask @captainpete to revert here?